docs/TESTING.md

```diff
@@ -0,0 +1,15 @@
+# Setup
+
+Install some dependencies before getting started:
+
+1. Run `npm install`
+2. Install a standalone Redis server
```

Comment:
Nice catch. I'm planning to do that in a future PR. Wanted to keep this one just about Postgres.